integration/tests/Puppetfile/HTTP_PROXY_effects_git_source.rb

```diff
@@ -19,7 +19,7 @@
 
 puppetfile =<<-EOS
 mod 'motd',
-  :git    => 'https://github.com/puppetlabs/puppetlabs-motd'
+  :git    => 'git://github.com/puppetlabs/puppetlabs-motd'
```

Comment:
Does this still test the same thing? iow, Does it have to be https to test the proxy?

Reply:
it looks like it's expecting failure below but with a failure that it could use the proxy correctly. I wonder why it's even falling back to github if the proxy doesn't work?

Reply:
Ahh, I don't think this was failing:
https://cinext-jenkinsmaster-enterprise-prod-1.delivery.puppetlabs.net/job/experimental_auto_r10k_intn-van-sys_irving/GIT_PROVIDER=shellgit,SLAVE_LABEL=beaker,TEST_TARGET=sles11-64.mdca/2/#showFailuresLink

not a big deal, and doesn't hurt to change it, was just curious

Reply:
Oh, whoops. I'll put this one and the other proxy one back. Thanks for the catch!

Reply:
Going to run both versions first, see if it made any difference.